<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="FetchAllFlights">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>All Flights</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fetchFlightInfo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>510</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Flight Info</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetFlights">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>721</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FlightRegister">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetFlightInfo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>331</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
